### ISSUE TITLE
Naming scheme for incompressibles

### ISSUE THREAD

**Issue by jowr** [CoolProp/CoolProp#85]
Any suggestions for a new way of naming the incompressibles? My random abbreviations start to become a little confusing when looking at a list of 100 fluids...

**Comment by jowr** [comment]
I agree with the aliasing suggestion. In general, I think we should go for one short-hand notation and something more intuitive and legible. A third alias could the be the old name to make the change backwards-compatible.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
